Rocket.Chat/NewRoomViewController.swift

```diff
@@ -193,6 +213,24 @@ extension NewRoomViewController: FormTableViewDelegate {
     func getPreviousValue(key: String) -> Any? {
         return setValues[key]
     }
+
+    func updateTable(key: String) {
+        tableView.beginUpdates()
+        tableView.endUpdates()
```

Comment:
What does these two lines make? 🤔

Reply:
Forces tableView to recalculate cells height with animation